app/ember/package.json

```diff
@@ -43,8 +43,9 @@
   "peerDependencies": {
     "@babel/core": "*",
     "babel-plugin-ember-modules-api-polyfill": "^2.12.0",
-    "babel-plugin-htmlbars-inline-precompile": "2 || 3",
-    "ember-source": "^3.16.0"
+    "babel-plugin-htmlbars-inline-precompile": "2 || 3 || 4",
```

Comment:
Based on the changelog, the only thing that has changed recently is node support

https://github.com/ember-cli/babel-plugin-htmlbars-inline-precompile/blob/master/CHANGELOG.md

Reply:
Can it work without the peer deps? If so, would prefer without them.

Reply:
I'm not entirely sure 😬 I'm happy to take direction on this 👀 

It is a dependency of `ember-cli-htmlbars`, so maybe that's enough?

Reply:
For me that's enough 😅